nltk/test/unit/lm/test_models.py

```diff
@@ -321,27 +321,31 @@ def setUp(self):
         self.model.fit(training_text)
 
     score_tests = [
-        # For unigram scores by default revert to MLE
+        # For unigram scores by default revert to Add-One Smoothed MLE
         # Total unigrams: 18
+        # Vocab Size = 7
         # count('c'): 1
         ("c", None, 1.0 / 18),
         # in vocabulary but unseen
         # count("z") = 0
-        ("z", None, 0.0 / 18),
+        ("z", None, 0 / 18),
         # out of vocabulary should use "UNK" score
         # count("<UNK>") = 3
         ("y", None, 3.0 / 18),
-        # gamma(['b']) = 0.1111
+        # 2 words follow b and b occured a total of 2 times
+        # gamma(['b']) = 2/(2+2) = 0.5
         # mle.score('c', ['b']) = 0.5
-        # (1 - gamma) * mle + gamma * mle('c') ~= 0.45 + .3 / 18
-        ("c", ["b"], (1 - 0.1111) * 0.5 + 0.1111 * 1 / 18),
+        # mle('c') = 1/18 = 0.055
+        # (1 - gamma) * mle + gamma * mle('c') ~= 0.27 + 0.055
+        ("c", ["b"], (1 - 0.5) * 0.5 + 0.5 * 1 / 18),
         # building on that, let's try 'a b c' as the trigram
-        # gamma(['a', 'b']) = 0.0667
+        # 1 word follow ab and ab occured 1 time
```

Comment:
```suggestion
        # 1 word follows 'a b' and 'a b' occured 1 time
```
This should make it consistent with 'a b c' on the previous line